LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
@bhavikapanara Why remove the LICENSE file?

Reply:
By mistake, it was removed. you can check in the current update.